### ISSUE TITLE
Error Decoder in Production mode

### ISSUE THREAD

**Issue by Gashik209** [subschema/subschema#54]
When build project using "create-react-app" and deploy it, we get error "Element type is invalid: expected a string (for built-in components) or a class/function (for composite components) but got: undefined. Check the render method of ObjectType"
When we using "create-react-app" in development mode - all OK.

src/App.js
```
import React, { Component } from 'react';
import schema from './schema.json'
import { Form, ValueManager } from 'subschema'
const errors = {}
const values = {}
const vm = ValueManager(values, errors)
class App extends Component {
  render() {
    return (
      <div className="App">
        <div id='flexSchema'>
          <Form schema={schema} valueManager={vm} />
        </div>
      </div>
    );
  }
}
export default App;
```

src/schema.json
```
{
  "schema": {
    "location": {
      "type": "Select",
      "title": "Location:",
      "options": [
        "Denver",
        "Cologne",
        "Houston",
        "Los Gatos"
      ]
    },
    "Name": {
      "type": "Text",
      "title": "Name:",
      "validators": [
        "required"
      ]
    },
    "Group": {
      "type": "Select",
      "title": "Max Group:",
      "options": [
        1,
        2,
        3,
        4
      ]
    }
  },
  "fieldsets": [
    {
      "legend": false,
      "fields": ["location", "Name", "Group"]
    }
  ]
}
```

**Comment by jspears** [comment]
Hrm... 
I copied pasta'd your schema into https://subschema.github.io/#Basic and it worked.

Do you know what version you are running?

**Comment by jspears** [comment]
I just ran 
[super-duper.tar.gz](https://github.com/subschema/subschema/files/1132515/super-duper.tar.gz)
```sh
$ tar -xvzf super-duper.tar.gz
$ cd super-duper
$ yarn install
$ yarn run start
```
and it worked for me.

**Comment by jspears** [comment]
doh, did see the production mode thing.
sorry.

**Comment by Gashik209** [comment]
```
  "dependencies": {
    "react": "^15.6.1",
    "react-dom": "^15.6.1",
    "react-scripts": "1.0.10",
    "subschema": "^2.2.3"
  }
```

npm start -> all work fine (dev mode)
npm run build -> cd ./build -> python -m http.server 80 -> have error (prod mode)
Look like some problems when we use:
```
      "env": {
        "NODE_ENV": "production",
      }
```

**Comment by jspears** [comment]
Ah, minification...
My guess is I am missing a displayName on a class... and that is causing this.  I'll have to do a bit more digging.

**Comment by Gashik209** [comment]
I use npm 4.2.0 and node v7.8.0
Did you get the same error like my?

**Comment by jspears** [comment]
So I think I have it figured out any luck I'll do a release tomorrow.

Thanks so much for reporting this.

**Comment by Gashik209** [comment]
Cool, thanks!

One more question:
by default subschema for labels use className ".col-sm-2" and for input(select) ".col-sm-10". How can I get className ".col-sm-4" for all labels and ".col-sm-8" for all FormControls in subschemas form?

**Comment by jspears** [comment]
Fixed in v2.2.9
==
Need to update docs for custom resolver must start using PropTypes from subschema-prop-types, otherwise this problem will happen again.

**Comment by jspears** [comment]
@Gashik209 
Cool, thanks!

One more question:
Yup, you can do a couple of things.
The best is to 
```
 import EditorTemplateStyle from 'subschema-css-bootstrap/lib/EditorTemplate-style';
Object.assign(EditorTemplateStyle, {
    label: "col-sm-4 control-label",
    error: 'error-block help-block',
    help: 'help-block',
    hasError: 'has-error',
    hasTitle: "col-sm-8",
    noTitle: "col-sm-12"
});
```
